### ISSUE TITLE
Add examples inside of https://github.com/ansys/example-data

### ISSUE THREAD

**Issue by SMoraisAnsys** [ansys/pyedb#51]
Currently, the examples leverage example data from `pyaedt` directory (see https://github.com/ansys/example-data/tree/master/pyaedt).
It would be nice to have a dedicated directory for edb.

Prerequisite: final name for this pakage.
